### ISSUE TITLE
内存泄漏(memory leak)

### ISSUE THREAD

**Issue by tanzhiwei2356** [SDWebImage/SDWebImage#3454]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name                               ios
 Platform Version                       16.0.1
 SDWebImage Version               5.14.2
 Integration Method              cocoapods 
 Xcode Version                   Xcode 14.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps
![截屏2022-12-05 16 52 16](https://user-images.githubusercontent.com/16899239/205598461-16bdde7b-da73-46c3-a69c-26fb26b84a49.png)

**Comment by dreampiggy** [comment]
You should click in and show me, what's line cause the leak, and what Types of object been leaked.
 
Your screenshot can not show any useful information.

**Comment by dreampiggy** [comment]
Emm... This is not leak, I think.

Just because of the decoding need temp allocation.

You can try to avoid huge image using the [Thumbnail Decoding](https://github.com/SDWebImage/SDWebImage/wiki/Advanced-Usage#thumbnail-decoding-550) feature

Or see here to disable the default-to-true Force Decoding feature: https://github.com/SDWebImage/SDWebImage/wiki/Common-Problems#optimization-for-large-images

**Comment by dreampiggy** [comment]
Please re-check with the latest v5.14.3 version. Which may reduce the JPEG/HEIF RAM usage by using Apple's YUV420 optimization
